Makefile

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/make -f
+
+PACKAGES=$(shell go list ./...)
+BUILDDIR ?= $(CURDIR)/build
+CURDIRR := $(CURDIR)
+COMMIT := $(shell git log -1 --format='%H')
+DOCKER := $(shell which docker)
+DOCKER_BUF := $(DOCKER) run --rm -v $(CURDIR):/workspace --workdir /workspace bufbuild/buf
+
+###############################################################################
+###                                Protobuf                                 ###
+###############################################################################
+
+proto-all: proto-gen proto-lint
+.PHONY: proto-all
+
+proto-gen:
+	@docker pull -q tendermintdev/docker-build-proto
+	@echo "Generating Protobuf files"
+	@docker run -v $(shell pwd):/workspace --workdir /workspace tendermintdev/docker-build-proto sh ./scripts/protocgen.sh
+.PHONY: proto-gen
+proto-lint:
+	@$(DOCKER_BUF) lint --error-format=json
```

Comment:
Decided to use similar proto generation to what tendermint uses instead of what the app/sdk/starport uses

Reply:
I am ok with that if this is easier, but feels like overkill.